src/aiParams.ts

```diff
@@ -59,7 +59,7 @@ export interface LangChainParams {
   openRouterModel: string;
   lmStudioBaseUrl: string;
   openAIProxyBaseUrl?: string;
-  useOpenAILocalProxy?: boolean;
+  enableCors?: boolean;
```

Comment:
Personally I'd prefer having this in CopilotSettings instead of LangChainParams since it has nothing to do with Langchain. But no worries, it will be address by my following PR. You can leave it here.

Reply:
It's here because it's needed while constructing `ProxyOpenAIEmbeddings` when it's constructed in `EmbeddingManager`. Are you considering adding the settings here? https://github.com/gianluca-venturini/obsidian-copilot/blob/ed96670e84aa2ab0243f1bdd81691fa30c767633/src/LLMProviders/embeddingManager.ts#L23

Reply:
@gianluca-venturini yes, in #567 I pass an array of `CustomModel` objects where each model has its own `enableCors` that the user can set in a table. I'll land your PR first and mine later.